### ISSUE TITLE
why loss have nan ,the loss function is smooth L1, when change the backbone from ResNet to ResNeSt?

### ISSUE THREAD

**Issue by zhangyu13141** [zhanghang1989/ResNeSt#146]

**Comment by zhangyu13141** [comment]
When the network start training

**Comment by zhanghang1989** [comment]
What the batch-size per-gpu? Ideally, it should be 16 or greater. If not, please try using SyncBatchNorm.
The reason is that there is BatchNorm in the Spli-Attention module

**Comment by zhangyu13141** [comment]
4 or 5 batch-size per-gpu, using SyncBatcjNorm within the ResNeSt  or in the fusion feature extract from the backbone, 
I try using SyncBatchBorm in the fusion feature，but the loss still nan, then I change from 0 to 1e-10 in the bias initialization: nn.init.constant_(g.bias, 1e-10), the bias initialization in the feature fusion, the loss wasn't nan, may I change it in this way?

**Comment by zhanghang1989** [comment]
That's fine. Ideally, the model using BN should not be sensitive for initialization method.

**Comment by zhangyu13141** [comment]
Output of the predict map, the predict of height is -inf, the classification score is [[0.,0.,...]], the previous batchsize output is classification score is [[0.04,0.05...]], the height[[0.2,0.5,-0.6....]]
